python/doc/theory/data_analysis/kolmogorov_test.rst

```diff
@@ -1,87 +1,65 @@
-
-This distance may be expressed as follows:
-
-.. math::
-
-    D = \sup_x \left|\widehat{F}_N\left(x\right) - F\left(x\right)\right|
-
-With a sample :math:`\left\{ x_1,x_2,\ldots,x_N \right\}`, the distance is estimated by:
+This test involves the calculation of the test statistic which is the weighted maximum
+distance between the empirical cumulative distribution function
+:math:`\widehat{F}` built from the sample and :math:`G`.
+Letting :math:`X_1, \ldots , X_\sampleSize` be i.i.d. random variables following
+the distribution with CDF :math:`F`, the test statistic is defined by:
 
 .. math::
 
-    \widehat{D}_N = \sup_{i=1 \ldots N}\left|F\left(x_i\right)-\frac{i-1}{N} ; \frac{i}{N}-F\left(x_i\right)\right|
+    D_{\sampleSize} = \sqrt{\sampleSize} \sup_{1 \leq i \leq \sampleSize} \left|\widehat{F}\left(X_i \right) - G\left(X_i \right)\right|
 
-Assume that the sample is drawn from the candidate distribution.
-By definition, the *p*-value of the test is the probability:
+The empirical value of the test statistic, evaluated from the sample is:
 
 .. math::
 
-    p = P(D > \widehat{D}_N)
-
-In the case where the fit is good, the value of :math:`\widehat{D}_N` is
-small, which leads to a p-value closer to 1.
-The candidate distribution will not be rejected if and only if
-:math:`p` is larger than a given threshold probability.
-In general, the threshold p-value is chosen to be 0.05:
-
-.. math::
-
-    p_{ref} = 0.05
-
-Based on the p-value,
-
-- if  :math:`p<p_{ref}`, we reject the candidate distribution,
-
-- otherwise, the candidate distribution is not rejected.
-
-Two situations may occur in practice.
-
-- the parameters of the distribution under test are known,
-
-- the parameters of the distribution under test are estimated from a sample.
-
-If the parameters of the distribution under test are known,
-algorithms are available to directly compute
-the distribution of :math:`\widehat{D}_N` both for *N* large
-(asymptotic distribution) or for *N* small (exact distribution).
-This is because the distribution of :math:`\widehat{D}_N` does
-not depend on the candidate distribution.
-
-If the parameters of the distribution under test are estimated
-from a sample, the :math:`\widehat{D}_N` statistic is generally smaller, because
-the parameters of the distribution have been computed from the
-sample.
-In general, the distribution of :math:`\widehat{D}_N` is not known
-and depends on the candidate distribution.
-Therefore, sampling methods can be used in order to estimate the p-value.
-
-The diagram below illustrates the principle of comparison with the empirical
-cumulative distribution function for an ordered sample
-:math:`\left\{5,6,10,22,27\right\}`; the candidate distribution considered here
-is the Exponential distribution with parameters :math:`\lambda = 0.07`,
-:math:`\gamma = 0`.
+    d_{\sampleSize} = \sqrt{\sampleSize} \sup_{1 \leq i \leq \sampleSize} \left|\widehat{F}\left(x_i\right) - G\left(x_i\right)\right|
```

Comment:
Same here: the sup is on i, not x_i

Reply:
I removed the sum and let the notation sup_x.